cylc/flow/etc/tutorial/cylc-forecasting-workflow/etc/python-job.settings

```diff
@@ -8,5 +8,5 @@
             # These environment variables ensure that tasks can
             # run in the same environment as the workflow:
             {% from "sys" import path, executable %}
-            PYTHONPATH = {{':'.join(path)}}
+            CYLC_PYTHONPATH = {{':'.join(path)}}
```

Comment:
I don't think this is right. The lib/python dir is for use by task scripts, not Cylc itself.

Reply:
This line was removed, then added back in again. I can see that this gets the tests passing, but I don't think it makes sense so may be highlighting a valid bug.

The `CYLC_PYTHONPATH` variable should only impact the Cylc application, but this is the environment section for a job. If `CYLC_PYTHONPATH` is required here, then `module load` commands won't work (because they set `PYTHONPATH`).

Reply:
This file deliberately describes itself as an anti-pattern - we're going out of our way to use the Cylc environment, not trying to avoid it.